geometric_kernels/kernels/matern_kernel.py

```diff
@@ -24,6 +24,7 @@
     DiscreteSpectrumSpace,
     Graph,
     Hyperbolic,
+    Hypercube,
```

Comment:
Maybe call this HypercubeGraph instead of just Hypercube?

Reply:
Ok, agree